components/generic/ThematicBreak/ThematicBreak.tsx

```diff
@@ -0,0 +1,3 @@
+export const ThematicBreak = () => {
+  return <hr className="w-full border-dashed border-gray-400"></hr>;
```

Comment:
It's not a thematic break, it's simply a border below the heading that's used for styling purposes.

Reply:
It's being reused throughout the project though, which is why it makes sense to extract it. What naming do you propose?

Reply:
Add it to heading as an option, it may good enough for us.